more_complex/README.md

```diff
@@ -0,0 +1,42 @@
+# A more Complex Example
+
+This example is designed to show you how to lay out a directory that can be
+used in Packer builds. Organizing related builds within directories makes it
+easier to reuse code and use variables. Here we split our build into several
+files:
+
+1. "init.pkr.hcl" describes all of our required plugins, required Packer
```

Comment:
```suggestion
1. `init.pkr.hcl` describes all of our required plugins, required Packer

Reply:
Consider adding this for the other lines, as well

Reply:
done